app/views/posts/index.html.erb

```diff
@@ -1,14 +1,18 @@
-<p style="color: green"><%= notice %></p>
 
-<h1>Posts</h1>
 
-<div id="posts">
-  <% @post.each do |post| %>
+
+<%= render partial: "users/user" , locals: { user: @user } %>
+ 
+<div class="post-container">
+  <% @posts.each_with_index do |post, index| %>
     <%= render post %>
-    <p>
-      <%= link_to "Show this post", post %>
-    </p>
-  <% end %>
-</div>
+    
+
+      <ul>
+        <% post.comments.each do |comment| %>
+          <%= render comment %>
```

Comment:
- [ ] Kindly note that on this page only you should preview only the most recent_comments(5 comments) instead of all comments. Hint `post.most_recent_five_comments`